code/index.html

```diff
@@ -1,30 +1,203 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+<head>
+	<meta charset="utf-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>International Cat Show by Ekaterina Klimenko</title>
+	<link rel="stylesheet" href="./style.css">
+	<link rel="preconnect" href="https://fonts.gstatic.com">
+	<link href="https://fonts.googleapis.com/css2?family=Cabin&display=swap" rel="stylesheet">
+	<link rel="preconnect" href="https://fonts.gstatic.com">
+	<link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;300;400&display=swap" rel="stylesheet">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+<body>
+	<!-- Hero Image section -->
+	<header class="hero-section">
+		<!-- Hero picture -->
+		<picture class="hero-picture">
+			<source srcset="./background_tablet.jpg" media="(min-width:768px) and (max-width: 949px)">
+			<source srcset="./background_desktop.jpg" media="(min-width: 950px)">
+			<img class="hero-image" src="./background_phone.jpg" alt="Cat picture" />
+		</picture>
+		<!-- Hero text -->
+		<div class="welcome-text">
+			<div class="welcome-text-inner">
+				<h1 class="hero-title">2021 International Cat Show</h1>
+				<h2 class="hero-subtitle">15 September 2021</h2>
+				<h2 class="hero-subtitle">Artipelag, Värmdö</h2>
+				<div class="welcome-buttons-wrapper">
+					<a class="button-wrapper" href="#form-link" class="welcome-button">Enter competition</a>
+					<a class="button-wrapper" href="" class="welcome-button">Buy tickets</a>
```

Comment:
There is really no reason to set `href` attribute without value - if you don't need it from practical perspective, just set it to `#` :
```
href="#"
```